tests/test_modeling_tf_electra.py

```diff
@@ -248,3 +248,19 @@ def test_model_from_pretrained(self):
         for model_name in ["google/electra-small-discriminator"]:
             model = TFElectraModel.from_pretrained(model_name)
             self.assertIsNotNone(model)
+
+
+class TFElectraModelIntegrationTest(unittest.TestCase):
+    @slow
+    def test_inference_masked_lm(self):
+        model = TFElectraForPreTraining.from_pretrained("lysandre/tiny-electra-random", use_cdn=False)
```

Comment:
Will remove the `use_cdn=False` tomorrow once the CDN caching has caught up.

```suggestion
        model = TFElectraForPreTraining.from_pretrained("lysandre/tiny-electra-random")
```